source/actions3/actions/AMControlWaitAction.cpp

```diff
@@ -141,7 +141,7 @@ bool AMControlWaitAction::checkCurrentControlValue()
 {
     const AMControlInfo& setpoint = controlWaitInfo()->controlInfo();
 
-    qDebug() << control_->name() << " Current: " << control_->value() << " New: " << setpoint.value();
+	 //qDebug() << control_->name() << " Current: " << control_->value() << " New: " << setpoint.value();
```

Comment:
Technically this is something that's been in AMControlWaitAction for as long as I can remember. I sometimes comment it out when I'm working with them, because they can get in the way of other debugs (in this case the ones from my tests). Usually I uncomment it before I push, but I forgot this time. 
There is an argument to be made for this to be a Error Mon debug (or info maybe)?